### PR TITLE
Enable snapshots

### PR DESCRIPTION
**Pull request by armanbilge**
I remember, I had initially disabled these b/c publishing creds weren't set up. They are now, so it should work.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 val catsVersion = "2.7.0"
 
 ThisBuild / tlBaseVersion := "21.8" // TODO
-ThisBuild / tlCiReleaseBranches := Seq()
+ThisBuild / tlCiReleaseBranches := Seq("master")
```
Comment (bpholt) [inline build.sbt:4]: If/when I change the default branch name to `main`, we'll need to update this as well, right? (And then regenerate the GHA files?)

Comment (armanbilge) [reply build.sbt:4]: Yes, exactly. Actually, at that point it can be omitted, since it the default setting.

Comment (armanbilge) [reply build.sbt:4]: Btw, feel free to let this PR sit till the branch rename. Unless you need snapshots :)

Comment (bpholt) [reply build.sbt:4]: I updated the default branch name to `main` today, so let's omit this.

Comment (armanbilge) [reply build.sbt]: Should be good to go!

Comment (bpholt) [reply build.sbt]: Thanks!

**Pull request merged**